addons/dexie-export-import/src/import.ts

```diff
@@ -45,6 +45,12 @@ export async function importDB(exportedData: Blob | JsonStream<DexieExportJsonSt
   return db;
 }
 
+export async function peakImportFile(exportedData: Blob): Promise<Partial<DexieExportJsonStructure>> {
+  const CHUNK_SIZE = (DEFAULT_KILOBYTES_PER_CHUNK * 1024);
+  const stream = await loadUntilWeGotEnoughData(exportedData, CHUNK_SIZE);
```

Comment:
`loadUntilWeGotEnoughData` imho does too much here and doesn't allow to handle file problems/errors on your own. It has two responsibilities: Read the header AND validate the file. My simple suggestion is to take the upper part of `loadUntilWeGotEnoughData`, i.e. the part before validation.

So I'd replace this line by:

```
  const stream = ('slice' in exportedData ?
    JsonStream<DexieExportJsonStructure>(exportedData) :
    exportedData);

  while (!stream.eof()) {
    await stream.pullAsync(CHUNK_SIZE);

    if (stream.result.data && stream.result.data!.data)
      break;
  }
```